cmd/device/list/list.go

```diff
@@ -23,22 +23,27 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/metadata"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/urlclient/local"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
 
 	"github.com/spf13/cobra"
 )
+
 const deviceTempl = "Device ID\tDevice Name\tOperating State\tAdmin State\tDevice Service\tDevice Profile\n" +
 	"{{range .}}" +
 	"{{.Id}}\t{{.Name}}\t{{.OperatingState}}\t{{.AdminState}}\t{{.Service.Name}}\t{{.Profile.Name}}\n" +
 	"{{end}}"
 
+var name string
+
 // NewCommand returns the list device command
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
```

Comment:
should you add in the doc string on line 41  “list  [<id> | --name/-n <name>] like you did for device services?

Reply:
Usually I use this addson "t [ | --name/-n ]" with remove commands. 
"List" commands usually have too much possible ways flags ...